src/functions.stub.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+  * @generate-function-entries
```

Comment:
The asterisk here and on the line below are indented an extra space. Looks like this was done in several other stub files, so you may want to just correct it all now.

Alternatively, this can probably just be a single line like some of the `@param` lines below, since you're not specifying `@generate-class-entries static` here.

Reply:
Ah, yeah. I fixed those for method comments in the original PR, but forgot the file docblock. I fixed them 👍